index.html

```diff
@@ -335,19 +336,19 @@ <h2>
           <ol class="algorithm">
             <li>Let |promise:Promise| be [=a new promise=].
             </li>
-            <li>When a window is in focus, if the [=this=]'s [=shadow-including
-            root=] is the [=navigable/active document=] of a [=Document/browsing
-            context=] (or has an [=tree/ancestor=] [=browsing context=]) which is not in
-            focus:
+            <li>When a {{Window/window}} is in [=Window/focus=], if the [=this=]'s
+            [=shadow-including root=] is the [=navigable/active document=] of a
+            [=Document/browsing context=] (or has an [=tree/ancestor=]
+            [=browsing context=]) which is not in focus:
               <ol>
                 <li>[=Queue an element task=] on the [=user interaction task
                 source=], given [=this=], to perform the following steps:
```

Comment:
```suggestion
                source=], given [=this=]'s [=relevant global=], to perform the following steps:
```

Reply:
I am not sure whether we need to add  [=relevant global=] after [=this=], because in https://html.spec.whatwg.org/multipage/webappapis.html#queue-an-element-task, it is expecting "element" as input then it use element's relevant global object internally inside it's algorithm.

Reply:
oh! you are right! My bad... I forgot [=this=] is an element.